src/schema/v2/__tests__/artists.test.ts

```diff
@@ -74,6 +96,189 @@ describe("Artists", () => {
     `
 
     const { artists } = await runQuery(query, { artistLoader })
+
     expect(artists[0].slug).toEqual("andy-warhol")
+    expect(artists[1].slug).toEqual("pablo-picasso")
+  })
+})
+
+fdescribe("artistsConnection", () => {
```

Comment:
(fdescribe)

Reply:
I do this all the time. Going to enable https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-focused-tests.md 😅